docs/providers.rst

```diff
@@ -1485,6 +1485,54 @@ The OpenID provider can be forced to operate in stateless mode as follows::
                     stateless=True,
                 )]}}
 
+
+OpenID Connect
+--------------
+
+The OpenID Connect provider provides a dynamic instance for each configured
+server. To expose an OpenID Connect server as an authentication method,
+configuration of one or more servers is required:
+
+.. code-block:: python
+
+    SOCIALACCOUNT_PROVIDERS = {
+        'openid-connect': {
+            'SERVERS': [
+                dict(
```

Comment:
It would be more readable to stick to one way of defining structures (so `{` instead of `dict(`).

Reply:
I have replaced `dict()` constructor usage with `{}` literals.